.github/workflows/cron.yml

```diff
@@ -63,3 +64,9 @@ jobs:
         run: |
           export PYTHONPATH=$(pwd):$PYTHONPATH
           pipenv run scrapy combinefeeds -s LOG_ENABLED=False
+
+      - name: Prevent workflow deactivation
+        uses: gautamkrishnar/keepalive-workflow@v1
+        with:
+          committer_username: "citybureau-bot"
+          committer_email: "documenters@citybureau.org"
```

Comment:
Should we use another email address, like `city-scrapers@citybureau.org`?
`documenters@citybureau.org` is more specific for Documenters project.

Reply:
Hmm, interesting question, @LienDang. I'm not sure if there is a `city-scrapers@citybureau.org` email address. There might be benefits to creating one but as I understand it we've historically always used documenters@citybureau.org for city-scrapers stuff. Eg. if you look at the city-scrapers project website:
![image](https://github.com/City-Bureau/city-scrapers-template/assets/37225902/f95aa586-3146-46d1-aee5-be8672d1fed3)
I would suggest we just go with this for now. A city-scrapers specific email address might be an item for another conversation.

Reply:
Agree with you on using a valid mailbox to accept inquiries. In the [contact section](https://cityscrapers.org/docs/next-steps/#contact-us) I found that cityscrapers@citybureau.org is used, not sure if it's valid, can you double check with CB team?

The source code is in city-scrapers repo (mainly used for Chicago)
https://github.com/City-Bureau/city-scrapers/blob/main/docs/_docs/05-nextsteps.md#contact-us
> Was this walkthrough helpful for you? Our team would love to know! Contact us any time at cityscrapers@citybureau.org, and please do let us know if you plan to, or have, built a City Scrapers network for your city.

Reply:
Ah, hmmm, I hadn't seen that email. I will check with Max on this.